addons/gsri_equipment/weapons_compat.hpp

```diff
@@ -0,0 +1,17 @@
+class Rifle;
+class Rifle_Base_F : Rifle {
+	class WeaponSlotsInfo;
+};
+class arifle_MSBS65_base_F : Rifle_Base_F {
+	magazineWell[] += {"CBA_556x45_STANAG"};
```

Comment:
Is your new MSBS compatible with both 6.5mm and 5.56mm ?

Reply:
Yes, I only add the 5.56 mag well to the weapon, not taking existing ones out.